api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -67,6 +67,20 @@ service StellarStationService {
   // If the satellite is not found or the client is not authorized for it, the stream will be closed
   // with a `NOT_FOUND` error.
   rpc OpenSatelliteStream (stream SatelliteStreamRequest) returns (stream SatelliteStreamResponse);
+
+  // Reserves a pass for later execution as a plan. The pass must be specified by a
+  // `reservation_token` returned in another method like `ListUpcomingAvailablePasses` and must be
```

Comment:
'and must be reservable' made me think that we're not allowed to call the method until we've checked whether the pass is reservable. Not sure if other people would read it that way though.

Reply:
I'm not very excited about it but my attempt to make it clearer isn't going so well :) I have the following text. Basically, it's a bit messy that the same error code is used for different error types (AOS too close, schedule conflict).

// Reserves a pass for later execution as a plan. The pass must be specified by a
// `reservation_token` returned in another method like `ListUpcomingAvailablePasses`.
//
// If the `reservation_token` is invalid, the request will return an `INVALID_ARGUMENT` error.
//
// If the requested pass cannot be scheduled due to a conflict with existing plans or other
// scheduling constraints, the request will return a `FAILED_PRECONDITION` error. This includes
// if the pass `aos_time` is less than one hour in the future.
//
// Note that the reservation_token may have become unreservable since it was fetched. If that's the case,
// the call to ReservePass will fail with a 'FAILED_PRECONDITION' error and `ListUpcomingAvailablePasses`
// should be called again to get a fresh list of reservable passes. The pass must be scheduled at least
// one hour in advance of `aos_time` or it will be rejected.
//